MongoConsumerLibary/MongoConnection/Collections/StatisticsCollection.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace MongoConsumerLibary.MongoConnection.Collections
+{
+    public class StatisticsCollection : ExpirableCollection
```

Comment:
why use in inheritance, put it in one class

Reply:
It is needed for the generic class CollectionEntity